src/cocotb_tools/makefiles/simulators/Makefile.vcs

```diff
@@ -71,15 +74,15 @@ endif
 # Can't do this using an argument, we have to create a PLI table file
 # enabling write access to the design
 $(SIM_BUILD)/pli.tab : | $(SIM_BUILD)
-	echo "acc+=rw,wn:*" > $@
+	echo "acc+=rw,wn,cbk:*" > $@
```

Comment:
Is there something akin to "all" we could use here instead?

Reply:
Do not see such options. There is global one (not recommended because slow) `+acc+3 ` that can be used instead of `pli.tab`. Maybe better in this case?

Reply:
Changed to global `+acc+3`

Reply:
Did you? I don't mind either way (and prefer the faster option, obviously), but I wanted to make sure the update didn't fall through the cracks.

Reply:
I think `all` or similar is a little better so we don't end up missing something we aren't currently checking, or something that may be fixed in a future version. The Makefiles are intended to be as usable as possible, not as fast as possible.